CHANGELOG.md

```diff
@@ -1,6 +1,4 @@
-# Unreleased Changes
-
-## Suggested release: v5.0.0
+## v5.0.0
 
 #### New Resources:
 This release adds support support to Oneview Rest API version 200 (only hardware variant C7000), 300 and 500 (both two for the hardware variants C7000 and Synergy) to the new features:
```

Comment:
Rewrite:

> This release adds support support to Oneview Rest API version 200 (only hardware variant C7000), 300 and 500 (both two for the hardware variants C7000 and Synergy) to the new features:

as:

_This release adds support to HPE OneView Rest API version 200 (for hardware variant C7000), 300 and 500 (both for the hardware variants C7000 and Synergy) to the new features:_